src/editor/editor.js

```diff
@@ -268,6 +270,34 @@ export default class Editor {
 		this.commands.execute( ...args );
 	}
 
+	/**
+	 * Throws `editor-wrong-element` when checked element is `<textarea>`.
+	 *
+	 * @protected
+	 * @param {HTMLElement|String} sourceElementOrData The DOM element that will be the source for the created editor
+	 * or the editor's initial data.
+	 */
+	static _assertAllowedSourceElement( sourceElementOrData ) {
+		if ( isElement( sourceElementOrData ) && sourceElementOrData.tagName.toLowerCase() === 'textarea' ) {
```

Comment:
Why `toLowerCase()`?

Reply:
You're right, it's completely unnecessary here.